ushahidi/platform-client
# Adding number of unmapped posts to map-view

This pull request makes the following changes:
- Adds info about nb of unmapped posts in map-view
- Adds a link to time-line where unmapped posts are filtered
- Displays the unmapped posts in time-line when clicking on link

Test these changes by:
- Pull in a nb of tweets, 
- Check the left margin of map-view, a test with nb of unmapped post should be visible if more than 1 unmapped posts are present
- Click on link with nb of posts
- The link should send you to the time-line and only show the unmapped posts.

**Concerns:** 
1. Should we add the option of only showing unmapped posts to the filter-bar as well?
2. There is no easy way to unset the unmapped-filter right now, should we add that?

Fixes ushahidi/platform#1392 .

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/574)
